airbyte-integrations/connectors/destination-starrocks/src/test-integration/java/io/airbyte/integrations/destination/starrocks/StarrocksDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.starrocks;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.StandardNameTransformer;
+import io.airbyte.integrations.standardtest.destination.DestinationAcceptanceTest;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.sql.*;
```

Comment:
Please import individuals class and functions not *

Reply:
I've removed all * imports and added the metadata.yaml file. Is there anything else that needs to be done?